tests/argumentvalidation.py

```diff
@@ -5,13 +5,17 @@
 import os
 import subprocess
 
-def validate_arguments(builddir, args_list, expected):
+
+def validate_arguments(builddir, args_list, expected, set_docker_incorrect=False):
```

Comment:
How about `disable_dockerd` (or `disable_docker_daemon`) as a name for the argument?

Reply:
Done